### ISSUE TITLE
Compatibility with Laravel 9

### ISSUE THREAD

**Issue by marnickmenting** [vdhicts/cyberfusion-cluster-api-client#98]
When trying to upgrade project to Laravel v9, composer requirements conflicts arise:

I don't really know how to handle this?

```
 Problem 1
    - Conclusion: don't install laravel/framework v9.0.0-beta.1 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.3 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.4 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.5 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.1 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.2 (conflict analysis result)
    - vdhicts/cyberfusion-cluster-api-client[v1.24.0, ..., v1.43.1] require illuminate/support ^8.22 -> satisfiable by illuminate/support[v8.22.0, ..., 8.x-dev].
    - Only one of these can be installed: illuminate/support[v4.2.0-BETA1, ..., 4.2.x-dev, v5.0.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev], laravel/framework[v9.0.0-beta.1, ..., 9.x-dev]. laravel/framework replaces illuminate/support and thus cannot coexist with it.
    - Root composer.json requires laravel/framework ^9.0 -> satisfiable by laravel/framework[v9.0.0-beta.1, ..., 9.x-dev].
    - Root composer.json requires vdhicts/cyberfusion-cluster-api-client ^1.24 -> satisfiable by vdhicts/cyberfusion-cluster-api-client[v1.24.0, ..., v1.43.1].
```

**Comment by dvdheiden** [comment]
Aha! This package needs to be updated to Laravel 9 too. Will do asap.

**Comment by dvdheiden** [comment]
I've already prepared a PR, but I need to test it a bit more. Will do that tonight.

**Comment by marnickmenting** [comment]
No rush, we have more issues to solve before we can go through with L9 :)

**Comment by dvdheiden** [comment]
I've just released v1.44.0 in which Laravel 9 is supported. Good luck with the upgrade.

**Comment by marnickmenting** [comment]
Tested and it works. Thanks!
